### PR TITLE
Add Accuracy, Drain Time, Play Time, Spectate stubs

### PR DESCRIPTION
**Pull request by Pennek**
Resolves part of #3149

### CODE REVIEW

File: wiki/Gameplay/Drain Time/en.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Drain Time
```
Comment (MegaApplePi) [inline wiki/Gameplay/Drain Time/en.md]:
```suggestion
# Drain time
```

File: wiki/Gameplay/Play Time/en.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Play Time
```
Comment (MegaApplePi) [inline wiki/Gameplay/Play Time/en.md]:
```suggestion
# Play time
```

File: wiki/Gameplay/Spectate/en.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+Spectating
```
Comment (MegaApplePi) [inline wiki/Gameplay/Spectate/en.md]: are you trying to add a tag here? if so, it should be like this

```suggestion
tags:
  - spectating
```

File: wiki/Gameplay/en.md [hunk 4]
```diff
@@ -0,0 +1 @@
+<!--TODO: Add index-->
```
Comment (MegaApplePi) [inline wiki/Gameplay/en.md]: You could just add a bulleted list to the stubs here for now.

File: wiki/Gameplay/Drain Time/en.md (hunk 1 above)
Comment (Pennek) [reply wiki/Gameplay/Drain Time/en.md]: I thought first-level headers were fine to capitalize all the way through? want a bit of clearance on that

File: wiki/Gameplay/Spectate/en.md (hunk 3 above)
Comment (Pennek) [reply wiki/Gameplay/Spectate/en.md]: gotcha - that's what i was going for. Is there a reason for tagging this way here and another way in Tournament articles (like ive done)? or is the tournament way just outdated somehow?

File: wiki/Gameplay/Drain Time/en.md (hunk 1 above)
Comment (MegaApplePi) [reply wiki/Gameplay/Drain Time/en.md]: Article titles are now going to use sentence case. See #2580 for discussion.

File: wiki/Gameplay/Spectate/en.md (hunk 3 above)
Comment (MegaApplePi) [reply wiki/Gameplay/Spectate/en.md]: Can you elaborate it bit more, I'm not sure if I understood your question correctly.

File: wiki/Gameplay/Play_time/en.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+# Play time
+
+Playtime is the total duration of a [beatmap](/wiki/Beatmaps). This includes the lead-in time and the break time(s). This usually correlates with the beatmap's song length.
```
Comment (MegaApplePi) [inline wiki/Gameplay/Play_time/en.md]:
```suggestion
Play time, or *playtime*, is the total duration of a [beatmap](/wiki/Beatmaps). This includes the lead-in time and the break time(s). This usually correlates with the beatmap's song length.
```

File: wiki/Gameplay/Play_time/en.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
+# Play time
```
Comment (MegaApplePi) [inline wiki/Gameplay/Play_time/en.md]:
```suggestion
---
tags:
  - length
  - playtime
---

# Play time
```

File: wiki/Gameplay/Spectate/en.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
+---
+tags:
+  - spectating
```
Comment (MegaApplePi) [inline wiki/Gameplay/Spectate/en.md]:
```suggestion
  - spec
  - spectating
```

File: wiki/Gameplay/Drain_time/en.md [hunk 8]
```diff
@@ -0,0 +1,7 @@
+# Drain time
```
Comment (MegaApplePi) [inline wiki/Gameplay/Drain_time/en.md]:
```suggestion
---
tags:
  - drain length
---

# Drain time
```

File: wiki/Gameplay/Drain_time/en.md [hunk 9]
```diff
@@ -0,0 +1,7 @@
...
+
+The drain time, or *drain*, refers to the total duration of a [beatmap](/wiki/Beatmaps) that requires the player to be active. Drain time does not consider breaks and unmapped parts throughout a beatmap.
```
Comment (MegaApplePi) [inline wiki/Gameplay/Drain_time/en.md]:
```suggestion
Drain time, or *drain*, refers to the total duration of a [beatmap](/wiki/Beatmaps) that requires the player to be active. Drain time does not include breaks and unmapped portions of a beatmap.
```

File: wiki/Gameplay/Drain_time/en.md [hunk 10]
```diff
@@ -0,0 +1,7 @@
...
+
+Maps with very long drain times are typically referred to as marathon maps and require endurance beyond the norm in order to complete successfully.
```
Comment (MegaApplePi) [inline wiki/Gameplay/Drain_time/en.md]:
```suggestion
Beatmaps with long drain times (usually longer than 5 minutes) are typically referred to as marathon maps and require endurance beyond the norm in order to complete successfully.
```

**Comment by MegaApplePi** [comment]
I think the accuracy stub reads weirdly.. I'm going to sleep on it before trying to make suggestions unless someone else gets to it.

**Comment by Pennek** [comment]
had to force push to correct my master branch not being in sync with ppy/master

File: wiki/Gameplay/Accuracy/en.md [hunk 11]
```diff
@@ -0,0 +1,5 @@
+# Accuracy
+
+Accuracy defines how accurately a player is hitting [hit objects](/wiki/Hit_Objects) on a [beatmap](/wiki/Beatmaps). This value varies from from 0 to 100%, and fluctuates up and down while a beatmap is being played. This happens because accuracy only takes objects already played into account. Accuracy can be seen in the top-right corner while watching a replay or playing a beatmap.
```
Comment (MegaApplePi) [inline wiki/Gameplay/Accuracy/en.md]: I'm still uncertain about this paragraph. I think the second and third sentences should be removed, but I'm going to wait to see if someone else has anything to comment about this stub.

File: wiki/Gameplay/Accuracy/en.md [hunk 12]
```diff
@@ -0,0 +1,5 @@
+# Accuracy
```
Comment (MegaApplePi) [inline wiki/Gameplay/Accuracy/en.md]:
```suggestion
---
tags:
  - acc
---

# Accuracy
```

File: wiki/Gameplay/Spectate/en.md [hunk 13]
```diff
@@ -0,0 +1,13 @@
+---
+tags:
+    - spec
```
Comment (MegaApplePi) [inline wiki/Gameplay/Spectate/en.md]:
```suggestion
  - spec
```

File: wiki/Gameplay/Spectate/en.md [hunk 14]
```diff
@@ -0,0 +1,13 @@
...
+    - spectating
```
Comment (MegaApplePi) [inline wiki/Gameplay/Spectate/en.md]:
```suggestion
  - spectating
```

File: wiki/Gameplay/Accuracy/en.md (hunk 11 above)
Comment (Pennek) [reply wiki/Gameplay/Accuracy/en.md]: ah, i agree. this only applies to osu! standard

**Pull request merged**